dbusmock/templates/logind.py

```diff
@@ -269,6 +270,7 @@ def AddSession(self, session_id, seat, uid, username, active):
                        ('Kill', 'ss', '', ''),
                        ('Lock', '', '', ''),
                        ('SetIdleHint', 'b', '', ''),
+                       ('SetLockedHint', 'b', '', ''),
```

Comment:
Shouldn't this actually set the `LockedHint` property? If you are testing this functionality, how does this make sense?

Reply:
Happy new year to you, too! No wonder my test wasn't quite working; I guess that's my first face-palm of the year :)

Reply:
Yes, that looks fine. Want to augment tests/test_logind.py's `test_session()` to check `loginctl show-session`'s output before/after calling SetLockedHint() on the mock? that would make sure that the properties are the ones that the real loginctl expects.

Reply:
Okay, done :)

I also added a check for the `Lock()`/`Unlock()` methods, but as a separate commit in case those aren't necessary or correct.